pelletier-construction-group-nextjs-8-28-home-page-component

Ali-7s opened the pull request:
Satisfies #28 
* Created homepage component with placeholder text

taylorpapke reviewed (changes requested):
It looks like @JoyZhang2023 already created a default home page called page.tsx when she created the nav bar component.

I would say that we should either edit the page.tsx to include contents from index.tsx or redirect home page to use index.tsx. We will also need to make sure that placeholder text is displayed under the nav bar.

JoyZhang2023 reviewed (changes requested):
@Ali-7s When I run `npm run dev` and got below page instead of home page placeholder. To be consistent with other projects structure, I suggest to update page.tsx as home page placeholder.
![image](https://github.com/PelletierConstructionGroup/pelletier-construction-group-nextjs/assets/137982978/01bbeab7-a917-4f0e-b399-ee34cda426d1)

Ali-7s commented:
I deleted index.tsx and updated page.tsx to be the new placeholder homepage file @taylorpapke @JoyZhang2023 
It should show like this now:
<img width="1507" alt="image" src="https://github.com/PelletierConstructionGroup/pelletier-construction-group-nextjs/assets/22021615/4aa20fec-548c-422d-9e92-04a9cc72091e">
